### PR TITLE
Brdf fallback merge

### PR DESCRIPTION
**Pull request by ASVincent**
* Need to determine if it's band1 or Band1
* Confirm brdf_threshold and doy_average in lineage
* Run on sample input
* Validate LZF intermediate compression / default

### CODE REVIEW

File: swfo/brdf_fallback.py [hunk 1]
```diff
@@ -400,34 +427,104 @@ def __get_data(dat_filename, param, window):
     return np.nanstd(data, axis=0, ddof=1, keepdims=False)
 
 
-def concatenate_files(infile_paths, outfile, h5_info, doy):
+def _calculate_valid_bounds(mask: np.Array, transform: Affine) -> Dict:
```
Comment (passangd) [inline swfo/brdf_fallback.py]: change input variable to reflect numpy array (mask: numpy.array)

File: swfo/brdf_fallback.py [hunk 2]
```diff
@@ -708,72 +820,78 @@ def __get_clean_data(data_filename, data_key, data_window):
...
+    """
+    Returns band specifications for the BRDF fallback datasets
     """
-    Generates a metadata doc for a brdf fallback dataset attributes are derived
-    from the metadata of the source dataset; assume metadata reference is ga-stac-like
-    :param h5info:
-        A dictionary of day-of-year strings to h5 collection references
-    :param doy:
-        A string of the day of year being averaged for the tile
+    obs_bands = ['band1', 'band2', 'band3', 'band4', 'band5', 'band6', 'band7']
```
Comment (passangd) [inline swfo/brdf_fallback.py]: obs_bands = Capitalize 'band*'

**Review by passangd — APPROVED** [review]
all good.

File: swfo/brdf_fallback.py (hunk 1 above)
Comment (ASVincent) [reply swfo/brdf_fallback.py]: thanks :)

File: swfo/brdf_fallback.py (hunk 2 above)
Comment (ASVincent) [reply swfo/brdf_fallback.py]: thanks :)

**Pull request merged**